### ISSUE TITLE
Add event rotation

### ISSUE THREAD

**Issue by DarthSlider** [salesforce/sloop#118]
It would be great if sloop could delete logs older then X days.

**Comment by sana-jawad** [comment]
@DarthSlider we already have this option of ""max-look-back" Please refer https://github.com/salesforce/sloop/blob/master/pkg/sloop/server/internal/config/config.go#L82

**Comment by DarthSlider** [comment]
> @DarthSlider we already have this option of ""max-look-back" Please refer https://github.com/salesforce/sloop/blob/master/pkg/sloop/server/internal/config/config.go#L82

Are these settings configurable via ENV?

**Comment by DarthSlider** [comment]
So, I could use all this https://github.com/salesforce/sloop/blob/968799a319c04ae0cdabebef53eafafea51b0c9f/pkg/sloop/server/internal/config/config.go#L26 settings in sloop.json, am I?

**Comment by DarthSlider** [comment]
@sana-jawad 
Ok, looks like some settings at sloop.json are just ignored.

This is my config file:
  sloop.json: |-
    {
      defaultNamespace: "default",
      defaultKind: "Pod",
      defaultLookback: "1h",
      MaxLookback: 302400000000000,
      MaxDiskMb: 9216,
      WatchCrds: false,
      leftBarLinks: [
      ],
      resourceLinks: [
      ]
    }

Changing default namespace works, but changing MaxLoockback, MaxDiscMB or WatchCrd doesn't.

Also I`m 100% sure that sloop reading this file, because if I put incorrect values it fails to start.

**Comment by sana-jawad** [comment]
yes all the variables are configurable. They can be specified in config file as well as runtime arguments. The issue with above sloop.json file is the casing. I have set it to 1hr for example 
![image](https://user-images.githubusercontent.com/57542098/81437188-9b13c800-911f-11ea-9621-1d8a56f9fdc7.png)

The configs can be checked via UI as well. Click on Sloop Debug Menu which appears on the left side of Sloop Instance UI. 
![image](https://user-images.githubusercontent.com/57542098/81437304-cc8c9380-911f-11ea-9311-e566634a9878.png)

Then click Config from Menu 
![image](https://user-images.githubusercontent.com/57542098/81437335-d7dfbf00-911f-11ea-9280-3b0f013a75fd.png)

Then the configs would be shown. The ook back time is 1 hr here (the value is displayed in nanoseconds)
![image](https://user-images.githubusercontent.com/57542098/81437406-f776e780-911f-11ea-9e00-c16552ea2ee9.png)

**Comment by DarthSlider** [comment]
@sana-jawad 
Fixed to no avail.
Here is my config:
![image](https://user-images.githubusercontent.com/24406367/81443281-d349f380-917d-11ea-8bf7-3343ba023ec9.png)
and it says: "panic: failed to unmarshal /sloopconfig/sloop.json. error unmarshaling JSON: json: cannot unmarshal string into Go struct field SloopConfig.maxLookBack of type time.Duration"

Changing it to the integer values with propper casing doesn't help:
![image](https://user-images.githubusercontent.com/24406367/81443818-cc6fb080-917e-11ea-99ed-17a2696bf694.png)

![image](https://user-images.githubusercontent.com/24406367/81443786-bfeb5800-917e-11ea-942c-38cc63e9e297.png)

**Comment by sana-jawad** [comment]
@DarthSlider how are you running sloop? Try with run time arguments like:

![image](https://user-images.githubusercontent.com/57542098/81445631-19776680-912e-11ea-9be2-67933399d57c.png)

**Comment by DarthSlider** [comment]
@sana-jawad running it in k8s via provided helm chart.
sloop.json is defined via configmap.
Running it with args says "flag provided but not defined: -max-look-back 4h"
Here are my args:
![image](https://user-images.githubusercontent.com/24406367/81451412-1c557400-918d-11ea-9e30-2f0501b304f8.png)

Which is weird because it also says:
"-max-look-back duration
Max history data to keep (default 336h0m0s)"

But "docker run sloopimage/sloop:latest sloop -max-look-back 4h" seems to work.

**Comment by mallow111** [comment]
@DarthSlider thanks for reporting the issue,  we will put it on our backlog, but feel free to send out a PR with a fix.
